shared/flexible-classroom/quickstart/ios.mdx

```diff
@@ -28,10 +28,10 @@ The source code for Flexible Classroom is located in the Github [CloudClass-iOS]
    git checkout release/{VERSION}
    ```
 
-   For example, if you want to switch to the branch for version `2.7.1`, run the command:
+   For example, if you want to switch to the branch for version `2.8.0`, run the command:
 
    ```bash
-   git checkout release/2.7.1
+   git checkout release/2.8.0
    ```
 
    The following image shows how to see the latest release branch in the GitHub repository:
```

Comment:
The section below contains two screenshots with the old repo and a mention of the old repo (Place the `apaas-extapp-ios` project under the same directory with `CloudClass-iOS`, as shown:)